### PR TITLE
#1126 - Add code coverage support

### LINKED ISSUE #1126: Add code coverage support
## Background
Add code coverage support to be able measure code quality.

## Feature
New ability to measure current code coverage as one of QA metrics.

### PR DESCRIPTION
**Pull request by miroslavpojer**
- Added maven profile into parent pom.
- Added code-coverage section in README.md file.

Closes #1126

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +63,15 @@ former indicates in which application version the given database schema was intr
 
 [^1]: Semantic Versioning - https://semver.org/
 
+## How to measure code coverage
+```shell
+./mvn verify -Pcode-coverage
```
Comment (wajda) [inline README.md]: We strongly prefer conditional profile activation (e.g. via `-D`) over the `-P`.

And in general, I guess this is a generic enough build feature that had better be included into the [AbsaOSS/root-pom](https://github.com/AbsaOSS/root-pom) repository.

**Review by cerveada — DISMISSED** [review]

**Review by wajda — CHANGES_REQUESTED** [review]
The change is implemented in the _root-pom_
AbsaOSS/root-pom#5

**Comment by miroslavpojer** [comment]
> The change is implemented in the _root-pom_ [AbsaOSS/root-pom#5](https://github.com/AbsaOSS/root-pom/pull/5)

I will update this PR when new version of root-pom will be released.
Update:
- Done in commit 30320b0a59f4c2924653158fd2f8ef6a25d00062.

File: README.md (hunk 1 above)
Comment (miroslavpojer) [reply README.md]: Refactored in commit 30320b0a59f4c2924653158fd2f8ef6a25d00062.

**Review by wajda — APPROVED** [review]
LGTM

**Review by cerveada — APPROVED** [review]

**Pull request merged**